### PR TITLE
exclude evrclient and timingapi modules build in PCDS environment

### PR DESCRIPTION
**Pull request by baljamal**
evrclient and timingapi modules build will be excluded in PCDS environment. That will keep this module usable for both AD and PCDS.

### CODE REVIEW

File: configure/RELEASE.local
Comment (JJL772) [inline configure/RELEASE.local:34]: Any reason why this is just `pcds` and not `/reg/g/pcds`? Either way works though, I suppose.

**Review by JJL772 — CHANGES_REQUESTED** [review]

**Comment by JJL772** [comment]
ALso, let's wait for @mpdunning's approval before merging, since he is the maintainer.

**Comment by baljamal** [comment]
in the future we should use /cds/group/pcds instead of /reg/g/pcds, so pcds part of the path will remain

**Comment by mpdunning** [comment]
Looks great, thanks for these changes.

**Pull request merged**
